projectgenerator/gui/export.py

```diff
@@ -19,12 +19,16 @@
 """
 
 import os
+from PyQt5.QtCore import Qt
+
+from PyQt5.QtWidgets import QCompleter
```

Comment:
Can you rewrite these to the same style as below? They both do the same, just for consistency.

    from qgis.PyQt.QtWidgets import xxx